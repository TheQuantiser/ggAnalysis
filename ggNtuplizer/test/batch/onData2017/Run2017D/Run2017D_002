file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/AA8DCD8A-3A37-E811-A727-1866DAEEB0A8.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/3AAB4940-4237-E811-B546-B083FED18B9F.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/58E6CB5B-5537-E811-842B-1866DAEEB364.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/0C85F9FD-6937-E811-B78F-782BCB20FDE8.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/46D10BB4-8E37-E811-A64A-1866DAEB528C.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/BC3587CB-8037-E811-88F6-1866DAEEB344.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/CACF1E99-5037-E811-9536-1866DAEB40CC.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/0A9C6291-6737-E811-A464-90B11C0BC86F.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/2C37CEB4-5737-E811-B6CF-1866DAEB296C.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/08F57ADC-7037-E811-9423-0019B9CB020D.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/34210F0D-6337-E811-8B31-801844DF0C50.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/3CDBC303-9137-E811-8E5B-801844E560A4.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/64045389-4437-E811-8F61-90B11C0BCD74.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/C80BFFB9-7937-E811-A3DC-A4BADB22B39D.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/020616E9-7237-E811-8631-F8BC1234F308.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/E8694253-4C37-E811-B14B-549F3525BF58.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/4403E4D3-7037-E811-A7D5-1866DAEECFD0.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/5454E551-4037-E811-ABDE-1866DAEB3628.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/88FD0C7C-3E37-E811-8B15-842B2B42BCF6.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/56DAF7BA-5037-E811-9E75-001EC94A8EF1.root
