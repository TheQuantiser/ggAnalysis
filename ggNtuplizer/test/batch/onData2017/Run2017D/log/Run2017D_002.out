scorpion6.spa.umn.edu
